backend/urls.py

```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.views.generic import TemplateView
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", TemplateView.as_view(template_name="index.html")),
```

Comment:
Change this to:
```
re_path("", TemplateView.as_view(template_name="index.html")),
```

Reference PR: https://github.com/bmiczek/django-react-boilerplate/pull/2